Weather/ViewModel/DailyForecastCell.swift

```diff
@@ -7,15 +7,34 @@
 //
 
 import Foundation
+import UIKit
+import CoreGraphics
 
 class DailyForecastCell: TableViewCell<DailyForecast> {
-	
+  
+  private let borderColor = UIColor.parse(0x888888)
+  private let borderTickness: CGFloat = 1
+  
+  override func setUp() {
```

Comment:
Bilgi olarak dursun; 

Her UIView in draw() metodu vardır, genellikle BezierPath ile çizim yapılacak şeyler draw() metodunda yapılır. Draw metodu direk olarak çağırılmaz, view in draw metodunu tetiklemek için setNeedsDisplay() çağırırsın. 

`
class Cell: UITableViewCell {
    
    override func draw(_ rect: CGRect) {
        
    }
    
}
`

Birde awakeFromNib var, view ler için viewDidLoad gibi düşün. Cell içinde initial property lerini burada setleyebilirsin. Cell indeki bu metotları buralardan tetikleyerek çağırabilirsin süperinden.

Reply:
setUp() func ının üstüne 
/// 
yazarak comment yazınca direk func ın description ında gözüküyor. Ben alışmaya çalışıyorum öyle yazyazmayama baya temiz olur senin base class ların için

Reply:
/// This is fucking awesome constructer for tableview cell
func setUp() { 
}

gibi :]